<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Projector</class>
 <widget class="QWidget" name="Projector">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>763</width>
    <height>86</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>3</x>
     <y>2</y>
     <width>120</width>
     <height>18</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="pj_status_icon">
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="ressources.qrc">:/icons/img/ko.png</pixmap>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lblPj">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;PROJECTOR F32&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>148</x>
     <y>24</y>
     <width>545</width>
     <height>47</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>6</y>
      <width>537</width>
      <height>35</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QRadioButton" name="src_vga">
       <property name="text">
        <string>VGA</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="src_bnc">
       <property name="text">
        <string>BNC</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="src_dvi">
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>DVI</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="src_svideo">
       <property name="text">
        <string>S-VIDEO</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="src_hdmi">
       <property name="text">
        <string>HDMI</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="src_comp">
       <property name="text">
        <string>COMPOSITE</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="src_rgb">
       <property name="text">
        <string>YPbPr</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>15</y>
     <width>141</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
   <widget class="QPushButton" name="pj_mute">
    <property name="geometry">
     <rect>
      <x>-5</x>
      <y>4</y>
      <width>149</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>MUTE</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pj_unmute">
    <property name="geometry">
     <rect>
      <x>-5</x>
      <y>29</y>
      <width>150</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>UNMUTE</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>695</x>
     <y>15</y>
     <width>66</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
   <widget class="QPushButton" name="pj_pwOff">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>67</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>OFF</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pj_pwOn">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>5</y>
      <width>66</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>ON</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>763</width>
     <height>86</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(228, 255, 224);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <zorder>frame</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>groupBox</zorder>
  <zorder>groupBox_2</zorder>
  <zorder>groupBox_3</zorder>
 </widget>
 <resources>
  <include location="ressources.qrc"/>
 </resources>
 <connections/>
</ui>
